Tutorials/intro-to-timing/example2.py

```diff
@@ -0,0 +1,35 @@
+
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.qua import *
+from qm import SimulationConfig
+import matplotlib.pyplot as plt
+from configuration import *
+QMm = QuantumMachinesManager()
+
+with program() as prog:
```

Comment:
What is the idea behind these examples?
Right now it's a set of programs without any explanation...